content/en/blog/2023/sig-network-blixt-ebpf-rust-in-kubernetes.md

```diff
@@ -0,0 +1,133 @@
+- Having the Blixt control-plane be a reference implementation.
+- Exploring issues associated with the generic use of eBPF in Kubernetes.
+
+In support of those goals we have some more specific sub-goals we're actively
+working towards:
+
+- Support [GatewayClass][gwc] and [Gateway][gw], meeting [conformance
+  requirements][gwconf]
+- Support [UDPRoute][urt] and [TCPRoute][trt], meanwhile helping to develop the
+  conformance requirements for these APIs.
+
+We have some significant progress on the above in that we have a **basic**
+level of support for creating a `GatewayClass` and `Gateway` and then attaching
+`UDPRoute` and `TCPRoute` resources to them, with the underlying dataplane
+receiving corresponding configuration and traffic then flowing as expected. We
+emphasize the **basic**: as the project is still quite early on, and being
+developed in a highly iterative fashion. That said the fundamentals are there
+and you can try them out yourself on a local system using our [usage
+documentation][usage] and [sample configurations][samples]. You can see more
+about the project's [current status on the README.md][status] including the
+milestones and current progress.
+
+One thing that can't be overstated about this project is that it has been at the
+center of a lot of learning, community building and fun. We have maintained a
+policy with this project that it shall never be intended for production use
+cases which means development of the project is more of a sandbox and a safe
+space for people to learn and experiment. If any of this sounds interesting to
+you now is a great time to get involved!
+
+[gwapi]:https://github.com/kubernetes-sigs/gateway-api
+[gwc]:https://gateway-api.sigs.k8s.io/api-types/gatewayclass/
+[gw]:https://gateway-api.sigs.k8s.io/api-types/gateway/
+[gwconf]:https://gateway-api.sigs.k8s.io/concepts/conformance/
+[urt]:https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.UDPRoute
+[trt]:https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.TCPRoute
+[usage]:https://github.com/kubernetes-sigs/blixt#usage
+[samples]:https://github.com/kubernetes-sigs/blixt/tree/main/config/samples
+[status]:https://github.com/kubernetes-sigs/blixt#current-status
+[bliss]:https://github.com/kubernetes-sigs/blixt/issues/new/choose
+[bldis]:https://github.com/kubernetes-sigs/blixt/discussions
+
+## Getting involved
+
+If you're interested in networking, Rust, Linux, eBPF (or all of the above)
+there's a lot of opportunity here to learn and have fun. We invite you to jump
+right in on the [repository][repo] if that's your style, or reach out to us in
+the community: You can reach us on [Kubernetes Slack][k8slack] on the
+`#sig-network-gateway-api` channel as well as the `#ebpf` channel. Blixt is a
+topic of discussion at the [Gateway API community meetings][meets], and the
+monthly [SIG Network Code Jam][cjam] as well.
+
+Cheers, and happy holidays!
```

Comment:
What holiday? (you might want to give context for readers who don't know that mid-November is a holiday season)

Reply:
I'll just remove it.

```suggestion
```